pkg/helm/chartproxy/proxy_test.go

```diff
@@ -44,7 +58,7 @@ func TestProxy_IndexFile(t *testing.T) {
 		{
 			name:       "returned merged index file for all accessible helm repos",
 			indexFiles: []string{"testdata/azureRepoIndex.yaml"},
-			mergedFile: "testdata/azureRepoIndex.yaml",
+			mergedFile: "testdata/mergedAzureRepoIndex.yaml",
```

Comment:
why this change here? we do not apply any filter here.

Reply:
Need to add this file to fix failing unit test